coderd/rbac/scopes.go

```diff
@@ -103,6 +126,213 @@ var builtinScopes = map[ScopeName]Scope{
 		},
 		AllowIDList: []string{policy.WildcardSymbol},
 	},
+
+	// User scopes (read + write pair)
+	ScopeUserRead: {
+		Role: Role{
+			Identifier:  RoleIdentifier{Name: "Scope_user:read"},
+			DisplayName: "Read user profile",
+			Site: Permissions(map[string][]policy.Action{
+				ResourceUser.Type: {policy.ActionReadPersonal},
+			}),
+			Org:  map[string][]Permission{},
+			User: []Permission{},
+		},
+		AllowIDList: []string{policy.WildcardSymbol},
+	},
+
+	ScopeUserWrite: {
+		Role: Role{
+			Identifier:  RoleIdentifier{Name: "Scope_user:write"},
+			DisplayName: "Manage user profile",
+			Site: Permissions(map[string][]policy.Action{
+				ResourceUser.Type: {policy.ActionReadPersonal, policy.ActionUpdatePersonal},
+			}),
+			Org:  map[string][]Permission{},
+			User: []Permission{},
+		},
+		AllowIDList: []string{policy.WildcardSymbol},
+	},
+
+	// Workspace scopes (read + write pair)
+	ScopeWorkspaceRead: {
+		Role: Role{
+			Identifier:  RoleIdentifier{Name: "Scope_workspace:read"},
+			DisplayName: "Read workspaces",
+			Site: Permissions(map[string][]policy.Action{
+				ResourceWorkspace.Type: {policy.ActionRead},
+			}),
+			Org: map[string][]Permission{
+				ResourceWorkspace.Type: {{ResourceType: ResourceWorkspace.Type, Action: policy.ActionRead}},
+			},
+			User: []Permission{},
+		},
+		AllowIDList: []string{policy.WildcardSymbol},
+	},
+
+	ScopeWorkspaceWrite: {
+		Role: Role{
+			Identifier:  RoleIdentifier{Name: "Scope_workspace:write"},
+			DisplayName: "Manage workspaces",
+			Site: Permissions(map[string][]policy.Action{
+				ResourceWorkspace.Type: {policy.ActionRead, policy.ActionCreate, policy.ActionUpdate, policy.ActionDelete},
+			}),
+			Org: map[string][]Permission{
+				ResourceWorkspace.Type: {
+					{ResourceType: ResourceWorkspace.Type, Action: policy.ActionRead},
+					{ResourceType: ResourceWorkspace.Type, Action: policy.ActionCreate},
+					{ResourceType: ResourceWorkspace.Type, Action: policy.ActionUpdate},
+					{ResourceType: ResourceWorkspace.Type, Action: policy.ActionDelete},
+				},
+			},
```

Comment:
`Org` level does not make sense in `Scope`. Just keep it all at the site level.
Remove the `org` permissions here and the others